package.json

```diff
@@ -83,6 +85,7 @@
     "react-ga": "^1.0.12",
     "react-router": "^1.0.3",
     "react-select": "^0.9.1",
+    "react-wp-content-loader": "latest",
```

Comment:
Oh yea, I've been `npm link` them since I'm developing both at the same time. Should've changed this to "git://github.com/mmmavis/react-wp-content-loader.git".  Let me update my PR.